CHANGELOG.md

```diff
@@ -26,46 +26,9 @@
 All notable changes to this project will be documented in this file.
 
 ## Unreleased
-* ### ALL
```

Comment:
No need to delete these lines. They will be deleted when we do a release.

Reply:
Do I need to add them back in? Will it mess something up if they're deleted?

Reply:
Nevermind, the merge took care of that.